tijdschrift-voor-economische-en-sociale-geografie.csl

```diff
@@ -0,0 +1,239 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Tijdschrift voor economische en sociale geografie</title>
+    <id>http://www.zotero.org/styles/tijdschrift-voor-economische-en-sociale-geografie</id>
+    <link href="http://www.zotero.org/styles/tijdschrift-voor-economische-en-sociale-geografie" rel="self"/>
+    <link href="http://www.zotero.org/styles/arachnology" rel="template"/>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1111/(ISSN)1467-9663/homepage/ForAuthors.html" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <eissn>1467-9663</eissn>
+    <updated>2017-07-14T16:54:52+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en-US">
+    <date form="text">
+      <date-part name="month" suffix=" "/>
+      <date-part name="day" suffix=", "/>
+      <date-part name="year"/>
+    </date>
+  </locale>
+  <macro name="editor">
+    <names variable="editor" delimiter=", ">
+      <name and="text" initialize-with=". " delimiter=", "/>
+      <label form="short" strip-periods="false" prefix=", "/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" text-case="capitalize-first" strip-periods="true" prefix="("/>
+  </macro>
+  <macro name="author">
+    <names variable="author" vertical-align="baseline">
+      <name and="symbol" delimiter-precedes-last="never" initialize-with="." name-as-sort-order="first">
+        <name-part name="family" text-case="uppercase"/>
+      </name>
+      <et-al font-style="italic"/>
+      <label form="short" prefix=", "/>
+      <substitute>
+        <names variable="editor"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author" prefix="(">
```

Comment:
I'm done otherwise. (if we fix the parentheses issue, we should check if we can quickly fix arachnology too, since this is pretty bad)